Integrations/DomainTools_Iris/DomainTools_Iris.py

```diff
@@ -0,0 +1,646 @@
+    domain_status = domain_result.get('active')
+
+    proximity_risk_score = 0
+    threat_profile_risk_score = 0
+    threat_profile_threats = ''
+    threat_profile_evidence = ''
+
+    overall_risk_score = domain_result.get('domain_risk').get('risk_score')
+    risk_components = domain_result.get('domain_risk').get('components')
+    if len(risk_components):
+        proximity_data = get_threat_component(risk_components, 'proximity')
+        blacklist_data = get_threat_component(risk_components, 'blacklist')
+        if proximity_data:
+            proximity_risk_score = proximity_data.get('risk_score')
+        elif blacklist_data:
+            proximity_risk_score = blacklist_data.get('risk_score')
+        threat_profile_data = get_threat_component(risk_components, 'threat_profile')
+        if threat_profile_data:
+            threat_profile_risk_score = threat_profile_data.get('risk_score')
+            threat_profile_threats = ', '.join(threat_profile_data.get('threats', []))
+            threat_profile_evidence = ', '.join(threat_profile_data.get('evidence', []))
+
+    website_response = domain_result.get('website_response')
+    google_adsense = domain_result.get('adsense').get('value')
+    google_analytics = domain_result.get('google_analytics').get('value')
+    alexa_rank = domain_result.get('alexa')
+    tags = domain_result.get('tags')
+
+    registrant_name = domain_result.get('registrant_name').get('value')
+    registrant_org = domain_result.get('registrant_org').get('value')
+    domain_registrant_contact = {
+        'Country': domain_result.get('registrant_contact').get('country').get('value'),
+        'Email': ', '.join([x.get('value') for x in domain_result.get('registrant_contact').get('email')]),
+        'Name': domain_result.get('registrant_contact').get('name').get('value'),
+        'Phone': domain_result.get('registrant_contact').get('phone').get('value'),
+    }
+    registrant_contact = {
+        'Country': domain_result.get('registrant_contact').get('country'),
+        'Email': domain_result.get('registrant_contact').get('email'),
+        'Name': domain_result.get('registrant_contact').get('name'),
+        'Phone': domain_result.get('registrant_contact').get('phone'),
+    }
+    admin_contact = {
+        'Country': domain_result.get('admin_contact').get('country'),
+        'Email': domain_result.get('admin_contact').get('email'),
+        'Name': domain_result.get('admin_contact').get('name'),
+        'Phone': domain_result.get('admin_contact').get('phone'),
+    }
+    technical_contact = {
+        'Country': domain_result.get('technical_contact').get('country'),
+        'Email': domain_result.get('technical_contact').get('email'),
+        'Name': domain_result.get('technical_contact').get('name'),
+        'Phone': domain_result.get('technical_contact').get('phone'),
+    }
+
+    billing_contact = {
+        'Country': domain_result.get('billing_contact').get('country'),
```

Comment:
no need to get `registrant_contact` so many times, it is better to init it first `billing_contact = domain_result.get('billing_contact', {}` and then use it to get the required fields